### PR TITLE
openra: introduce a FreeBSD rc.d script

### PR DESCRIPTION
**Pull request by moviuro**
Requires screen

### CODE REVIEW

**Comment by Mailaender** [comment]
Canceling CI builds as this is just stored here and never used.

File: packaging/freebsd/openra [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+: ${openra_session="openra-session"}
+: ${openra_user="openra"}
+: ${openra_chdir="/usr/home/${openra_user}"}
+: ${openra_path="/usr/home/${openra_user}/OpenRA/OpenRA.Game.exe"}
+
+: ${openra_ServerName="Dedicated_Server"}
+: ${openra_Mod="ra"}
+: ${openra_ListenPort="1234"}
+: ${openra_ExternalPort="1234"}
+: ${openra_AdvertiseOnline="False"}
```
Comment (obrakmann) [inline packaging/freebsd/openra]: I'd set this to true by default. I guess not many people run a private dedicated server.

**Comment by obrakmann** [comment]
Otherwise, :+1: . I have no way to test this, so I'll take your word for it.

File: packaging/freebsd/openra [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+stop_cmd="${name}_stop"
+status_cmd="${name}_status"
+
+: ${openra_enable="NO"}
+: ${openra_session="openra-session"}
+: ${openra_user="openra"}
+: ${openra_chdir="/usr/home/${openra_user}"}
+: ${openra_path="${openra_chdir}/OpenRA/OpenRA.Game.exe"}
+
+: ${openra_ServerName="Dedicated_Server"}
```
Comment (Mailaender) [inline packaging/freebsd/openra]: The underscore is not required. See https://github.com/OpenRA/OpenRA/pull/8206

Comment (moviuro) [inline packaging/freebsd/openra]: It is because the quotes disappear in the command, see L55

**Comment by Mailaender** [comment]
Asking @ihptru to review as I also don't run *BSD.

**Comment by Mailaender** [comment]
Okay, taking your word for it.

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/1a55793bbad09ef4420df111aae3ab3dd84ced31...2fcf0ea7c19b8658a0ae56ae57a62fcc2d11b7a6)

**Pull request merged**